backend/controllers/feedController.js

```diff
@@ -159,4 +159,29 @@ const getUserInfo = async (req, res, next) => {
   }
 };
 
-module.exports = { getActivity, getLearnigsCount, getUserInfo };
+const getAllUsersInfo = async (req, res, next) => {
+  try {
+    const users = await User.findAll();
+    const userList = [];
+    await users.map(async ({ id, first_name, last_name, avatar_url }) => {
+      userList.push({
+        id,
+        first_name,
+        last_name,
+        avatar_url,
+      });
+    });
```

Comment:
- please avoid mapping. why don't you just select the fields directly in your query so you don't need to map the list :thinking: 
```
Student.findAll({ attributes: ['name', 'birthdate'] })
```